pkg/apis/pipeline/v1beta1/container_types.go

```diff
@@ -200,6 +200,14 @@ type Step struct {
 	StderrConfig *StepOutputConfig `json:"stderrConfig,omitempty"`
 }
 
+// OnErrorType defines a list of supported exiting behavior of a container on error
+type OnErrorType string
+
```

Comment:
Wondering if there's specific reason that we need to change from using the reference in the pkg/entrypoint/engrypointer.go:
```
	ContinueOnError = "continue"
	FailOnError     = "stopAndFail"
)
```

Reply:
In general we prefer a unified way to define the OnErrorType to reduce maintenance cost. And we can benefit from the compile time check by the typed string.

It's also a bit weird to me to add the entrypoint package as an extra dependency to v1 and v1beta1 packages just to refer to a constant.